### ISSUE TITLE
recv_window timing issue after updating to 2.2.0 2.1.10

### ISSUE THREAD

**Issue by arenddeboer** [tiagosiebler/bybit-api#151]
After updating to 2.2.0 I receive a timing issue and am unable to authenticate.
Reverting back to 2.1.10 makes this problem go away.

On 2.2.0 when doing a getApiKeyInfo:
```
getApiKeyInfo {
  ret_code: 10002,
  ret_msg: 'invalid request, please check your timestamp and recv_window param. req_timestamp: 1653737893 server_timestamp: 1653737454742 recv_window: 5000',
  ext_code: '',
  ext_info: '',
  result: null,
  time_now: '1653737454.742912'
}
```
I can prevent this error by passing ```restOptions: { disable_time_sync: true }``` 
I think the websocket connection suffers from the same core issue and gives an auth error: 
```
Websocket error due to 401 authorization failure. { category: 'bybit-ws', wsKey: 'linearPrivate' }
```
It looks like the server timestamp is in milliseconds and the local timestamp in seconds, could this be the problem ?

**Comment by tiagosiebler** [comment]
Thank you for raising this. That's quite an oversight on my part as the time-sync feature was disabled in all the integration tests (as the lingering time-sync timer doesn't give a clean exit). You're right, in the refactoring I did around a standard base rest client, I missed the format of the timestamp used here.

Just pushed a fix for this via #152. Should be on npm shortly on version 2.2.1 of the package. On my end all private REST endpoints as well as the ws client (except for spot markets, different issue) seems to be working again.

**Comment by arenddeboer** [comment]
Awesome, thanks so much for this lib and the quick response
